Expensify/App
# Change screen name of SignIn route to "Home"

### Details
See issue for more context.

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/161155

### Tests
- Verified that for an unauthenticated account, the SignIn page is still the first thing that loads (and that there are no issues when rendering the component.
- Verified that you can sign in and that you're taken to the chat hoe
- Verified that you can sign out and that you're taken back to the sign in page.
- Verified that the flow for validating a secondary login still works.

### QA Steps
1. Sign out of any accounts. On mobile/desktop, simply open the app. On web, navigate to the root URL.
2. Verify that sign in page renders.
3. Verify that you can log in and that it redirects you to the chat home.
4. Verify that you can sign out and that it redirects you back to the sign in page.
5. Validate a secondary login and verify that the validation link works as expected (see QA steps for https://github.com/Expensify/Web-Expensify/pull/30615 for additional details).

### Tested On


### Screenshots

#### Web
<img width="1792" alt="Screen Shot 2021-04-21 at 1 11 07 PM" src="https://user-images.githubusercontent.com/31285285/115500026-217ad880-a2a3-11eb-9100-c842a53c4a58.png">

#### Mobile Web
![simulator_screenshot_6036C218-CDD3-4957-BCBF-4DC2C7F47D44](https://user-images.githubusercontent.com/31285285/115500146-55ee9480-a2a3-11eb-93f7-9c07dcb87e00.png)


#### Desktop
<img width="1792" alt="Screen Shot 2021-04-21 at 1 12 35 PM" src="https://user-images.githubusercontent.com/31285285/115500113-45d6b500-a2a3-11eb-96b2-c9c1e36b749b.png">

#### iOS
![Simulator Screen Shot - iPhone 11 - 2021-04-21 at 13 07 15](https://user-images.githubusercontent.com/31285285/115499784-9c8fbf00-a2a2-11eb-8bcd-c30cea4d2e24.png)

#### Android
![Screenshot_1618981772](https://user-images.githubusercontent.com/31285285/115499884-d660c580-a2a2-11eb-851f-e7bd1789006a.png)


cc @NikkiWines
